features/sharding/api/src/main/java/org/apache/shardingsphere/sharding/api/config/ShardingRuleConfiguration.java

```diff
@@ -65,4 +65,9 @@ public final class ShardingRuleConfiguration implements DatabaseRuleConfiguratio
     private Map<String, AlgorithmConfiguration> auditors = new LinkedHashMap<>();
     
     private ShardingCacheConfiguration shardingCache;
+    
+    @Override
+    public boolean isEmpty() {
+        return tables.isEmpty() && autoTables.isEmpty() && bindingTableGroups.isEmpty() && null == defaultDatabaseShardingStrategy && null == defaultTableShardingStrategy;
```

Comment:
if tables and autoTables are empty, then bindingTableGroups must be empty

Reply:
Fixed, THX